includes/Core/Util/Reset.php

```diff
@@ -178,4 +192,35 @@ private function get_rest_routes() {
 			),
 		);
 	}
+
+	/**
+	 * Handles the reset admin action.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $nonce WP nonce for action.
+	 */
+	private function handle_reset_action( $nonce ) {
+		if ( empty( $nonce ) || ! wp_verify_nonce( $nonce, static::ACTION ) ) {
```

Comment:
Checking whether `$nonce` is empty is redundant, `wp_verify_nonce` does the same.
```suggestion
		if ( ! wp_verify_nonce( $nonce, static::ACTION ) ) {
```

Reply:
Sounds good to me. We do the same thing many other places though 😄  Will update.